aasemble/django/apps/mirrorsvc/models.py

```diff
@@ -27,6 +27,10 @@ def lookup_by_user(cls, user):
             return cls.objects.all()
         return cls.objects.filter(owner=user) | cls.objects.filter(extra_admins=user.groups.all())
 
+    @property
+    def snapshots_url(self):
+        # print(self.uuid)
+        return '/mirrorsvc/mirrorsets/%s/snapshots/' % (self.uuid)
```

Comment:
Django has a method for looking this up. It's django.core.urlresolvers.reverse(). You pass it the view_name (as defined in your urls.py) and lookup arguments. You should use it instead of hardcoding the url scheme here.

Look at Django's documentation for it. Let me know if you need help.